home/tests.py

```diff
@@ -1,3 +1,34 @@
-# from django.test import TestCase
+from django.contrib.auth.models import User
+from home.models import Profile, Question, Answer
+import pytest
 
-# Create your tests here.
+
+@pytest.mark.django_db
+class TestModels:
+
+    def test_add_profile(self):
+        profile = self.get_profile()
+        assert profile.user.id >= 0  # check if user saved successfully and get an id
+        assert profile.gender == 'U'  # default gender is U
+
+    def test_add_answer(self):
+        profile = self.get_profile()
+        question = Question.objects.first()
+        if profile and question:
+            answer = Answer(profile=profile, question=question, content='Test')
+            answer.save()
+            assert answer.id >= 0  # check if answer saved successfully and get an id
+
+    def get_profile(self):
```

Comment:
it would be much more useful to move the lines below into a classmethod on one of the models, it would also make more sense to test it then!